.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,115 @@
+name: Bug Report
+description: Create a detailed bug report to help us resolve the issue
+title: "[BUG] <Title> "
+labels: ["bug", "triage"]
+assignees: []
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report. Please provide as much detail as possible to help us investigate.
+
+  - type: input
+    id: version
+    attributes:
+      label: BioNeMo Framework Version
+      description: What version or commit hash of the framework are you using?
+      placeholder: e.g., commit-hash or v1.2.3
+    validations:
+      required: true
+
+  - type: input
+    id: docker-image-info
+    attributes:
+      label: Docker Image
+      description: If the issue occurred in a container, provide the docker image name. Visit [BioNeMo Framework NGC website](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara/containers/bionemo-framework/tags) for available images.
+      placeholder: e.g., nvcr.io/nvidia/clara/bionemo-framework:2.2
+    validations:
+      required: false
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Bug Description
+      description: Provide a clear and concise description of the bug
+      placeholder: |
+        Describe what happened and what you expected to happen.
+        Include any relevant context about when/where this occurs.
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Steps to Reproduce
+      description: Provide detailed steps to reproduce the behavior
+      placeholder: |
+        1. Configure environment with '...'
+        2. Run command '...'
+        3. Execute function '...'
+        4. See error
+
+        Code example:
+        ```python
+        # Minimal code that reproduces the issue
+        ```
+    validations:
+      required: true
```

Comment:
My only thought is that by making all these fields required, we're raising the burden to submit a github issue. Some issues likely don't need all this info (i.e., xyz resource doesn't exist, missing argument in CLI, etc.)

Reply:
I can mark as not required Error Messages and Logs and System Information and maybe removed Submission Checklist?

Reply:
Yeah whatever you think is reasonable, just wanted to point that out as a possible concern. Let's revisit if/when this is onerous to submit new trivial issues (xfailing a test from CI, etc.)